### ISSUE TITLE
[BB] rare-table-v4 seems to have Kondrieu and Shambertin drops swapped

### ISSUE THREAD

**Issue by toiletman74** [fuzziqersoftware/newserv#557]
**Describe the bug**
rare-table-v4 seems to have Kondrieu and Shambertin drops swapped or atleast the rates are swapped. Shambertin has a bunch of drops that are 13/128 (= ~1/10) and Kondrieu has a bunch of drops that are 7/512 (= ~1/73)

**Game version(s) (choose one or more of the following):**
BB

**Comment by nolrinale** [comment]
Yes they are swapped (both drops and rates) and I already reported this a long time ago and was decided on the spot that it was not going to be fixed due to "several conflicting information" regarding the EP4 drops not sure what conflict information is about as the drops for JPBB are available [here](https://psobb.xrea.tk/droplist/psobb_drop_list_ep4.html)   but that was the reason I was given and that will be left "as-is" for the time being.

Will it be fixed now?
